kits/eda-analyst/prompts/eda-analyst_instructor-llmnode-740_user_1.md

```
@@ -0,0 +1 @@
+Dataset profile (includes dataset-level stats, per-column profiles, and correlations):{{codeNode_458.output.profile}}Plan the analysis tasks.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)